### ISSUE TITLE
odata-client-runtime dependency on jaxb-core and jaxb-impl

### ISSUE THREAD

**Issue by dbaldes** [davidmoten/odata-client#209]
odata-client-runtime has the following dependencies:

```xml
        <dependency>
            <groupId>org.glassfish.jaxb</groupId>
            <artifactId>jaxb-runtime</artifactId>
            <version>3.0.2</version>
        </dependency>
        <dependency>
            <groupId>com.sun.xml.bind</groupId>
            <artifactId>jaxb-core</artifactId>
            <version>3.0.2</version>
        </dependency>
        <dependency>
            <groupId>com.sun.xml.bind</groupId>
            <artifactId>jaxb-impl</artifactId>
            <version>3.0.2</version>
        </dependency>
```

So it uses the "new" jaxb-runtime along with the "old" jaxb-core and jaxb-impl. As this caused conflicts with other libraries in our project, I excluded the com.sun.xml.bind dependencies, and the odata client seems to work just fine.

Why are the dependencies to the com.sun.* jaxb libraries needed?

**Comment by davidmoten** [comment]
good point, thanks, I'll sort it.

**Comment by davidmoten** [comment]
The fix is on Maven Central in 0.1.73 (0.1.72 had it but sonatype was having trouble so that version didn't make it up). Thanks for the report.
